src/views/Transactions.vue

```diff
@@ -1,34 +1,39 @@
 <template>
     <div class="transaction transaction_list" ref="txListElement">
-      <md-list class="custom-list md-triple-line md-transparent">
-        <md-list-item v-for="(transaction) in transactions" :key="transaction.id" style="cursor:pointer">
-          <md-avatar>
-            <md-icon>{{ transaction.direction === 'from' ? 'flight_takeoff' : 'flight_land' }}</md-icon>
-          </md-avatar>
 
-          <div class="md-list-text-container" v-on:click="goToTransaction(transaction.id)">
-            <div>
-              {{ displayName(transaction) }}
-              <span class="partner_display_name">{{ formatPartnerAddress(transaction) }}</span>
+      <template v-if="isFulfilled">
```

Comment:
Cannot we use `areTransactionsLoading` instead? Also, as far as I understand, this component will display nothing while `isFulfilled == false` - is this correct? I believe, some kind of a spinner should be displayed.